dockstore-integration-testing/src/test/resources/dockstore.yml

```diff
@@ -13,6 +13,10 @@ bitbucketClientSecret: <fill me in>
 discourseKey: <fill me in>
 discourseUrl: <fill me in>
 discourseCategoryId: 9
+zenodoClientID: <fill me in>
+zenodoClientSecret: <fill me in>
+zenodoRedirectURI: http://<fill me in>:8080/auth/zenodo.org
```

Comment:
This is typically going to be a url in the browser, isn't it? For example, http://localhost:4200/auth/zenodo.org?

In your [UI PR](https://github.com/dockstore/dockstore-ui2/pull/668/files#diff-d7234fe7e1d30bd4e0d4370cc19b3a2cR59), you take the zenodoRedirectPath from line 144 and prepend the hostname.

It would be nice to reduce this duplication, where you have `auth/zenodo.org` twice.

Reply:
Will clean up in a separate PR